kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala

```diff
@@ -109,6 +110,125 @@ private[v1] class SessionsResource extends ApiRequestContext {
     Response.ok().build()
   }
 
+  @GET
+  @Path("{sessionHandle}/executeStatement")
+  def executeStatement(@PathParam("sessionHandle") sessionHandleStr: String,
+                       @QueryParam("statement") statement: String,
+                       @QueryParam("runAsync") runAsync: String,
+                       @QueryParam("queryTimeout") queryTimeout: String): OperationHandle = {
+    checkQueryParams(statement, runAsync, queryTimeout)
+    val sessionHandle = getSessionHandle(sessionHandleStr)
+    try {
+      backendService.executeStatement(sessionHandle, statement, runAsync.toBoolean,
+        queryTimeout.toLong)
+    } catch {
+      case NonFatal(_) =>
+        throw new NotFoundException(s"Error executing statement")
+    }
+  }
+
+  @GET
+  @Path("{sessionHandle}/getTypeInfo")
+  def getTypeInfo(@PathParam("sessionHandle") sessionHandleStr: String): OperationHandle = {
+    val sessionHandle = getSessionHandle(sessionHandleStr)
+    try {
+      backendService.getTypeInfo(sessionHandle)
```

Comment:
Actually, this behavior is to create or submit a `getTypeInfo` operation, right?